src/dbnode/generated/proto/schematest2/otherpkg.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package schematest2;
```

Comment:
I don't really care personally, but if you like you can use the protoreflection library to parse the test schemas so you don't have to add all this generated code but w/e not a big deal

Reply:
I explicitly want to avoid using jhump's parse code. User should generate proto file descriptors using protoc in their repo already, our schema cli tool just pick it up from their generated go code.